src/fiatExchanges/FiatExchangeAmount.test.tsx

```diff
@@ -243,51 +243,40 @@ describe('FiatExchangeAmount cashOut', () => {
   })
 
   it('displays correctly for cUSD when local currency is USD', () => {
-    const { getByText, getByTestId } = render(
+    const { getByText } = render(
       <Provider store={storeWithUSD}>
         <FiatExchangeAmount {...mockScreenProps} />
       </Provider>
     )
     expect(getByText('amount (cUSD)')).toBeTruthy()
-    expect(getElementText(getByTestId('LineItemRowTitle/subtotal'))).toBe('cUSD @ $1.00')
-    expect(getElementText(getByTestId('LineItemRow/subtotal'))).toBe('$0.00')
-    expect(getByText('disclaimerFiat, {"currency":"cUSD"}')).toBeTruthy()
   })
 
   it('displays correctly for cEUR when local currency is USD', () => {
-    const { getByText, getByTestId } = render(
+    const { getByText } = render(
       <Provider store={storeWithUSD}>
         <FiatExchangeAmount {...mockScreenPropsEuro} />
       </Provider>
     )
     expect(getByText('amount (cEUR)')).toBeTruthy()
-    expect(getElementText(getByTestId('LineItemRowTitle/subtotal'))).toBe('cEUR @ $1.20')
-    expect(getElementText(getByTestId('LineItemRow/subtotal'))).toBe('$0.00')
-    expect(getByText('disclaimerFiat, {"currency":"cEUR"}')).toBeTruthy()
   })
 
   it('displays correctly for CELO when local currency is USD', () => {
-    const { getByText, getByTestId } = render(
+    const { getByText } = render(
       <Provider store={storeWithUSD}>
         <FiatExchangeAmount {...mockScreenPropsCelo} />
       </Provider>
     )
     expect(getByText('amount (CELO)')).toBeTruthy()
-    expect(getElementText(getByTestId('LineItemRowTitle/subtotal'))).toBe('CELO @ $5.00')
-    expect(getElementText(getByTestId('LineItemRow/subtotal'))).toBe('$0.00')
   })
 
-  it('displays correctly when the SHOW_RECEIVE_AMOUNT_IN_SELECT_PROVIDER feature flag is on', () => {
+  it('displays correctly', () => {
```

Comment:
This test is now identical to the `'displays correctly for cUSD when local currency is USD'` one, right? Can prolly remove this one now.

Reply:
Fixed in 0efe49eea5a1dd1a63affd881569a971f3912391!